pybliometrics/scopus/utils/get_content.py

```diff
@@ -77,7 +77,10 @@ def get_content(url, params={}, *args, **kwds):
         try:
             reason = resp.json()['service-error']['status']['statusText']
         except:
-            reason = ""
+            try:
+                reason = resp.json()['message']
+            except:
```

Comment:
One should not have a single `except`, which would catch all possible errors, but rather specify the precise exception that might arise. Otherwise one is at risk masking bugs.